x-pack/legacy/plugins/siem/public/components/inspect/translations.ts

```diff
@@ -13,3 +13,43 @@ export const INSPECT = i18n.translate('xpack.siem.inspectDescription', {
 export const CLOSE = i18n.translate('xpack.siem.inspect.modal.closeTitle', {
   defaultMessage: 'Close',
 });
+
+export const SOMETHING_WENT_WRONG = i18n.translate(
+  'xpack.siem.inspect.modal.somethingWentWrongDescription',
+  {
+    defaultMessage: 'Sorry about that, something went wrong.',
+  }
+);
+export const INDEX_PATTERN = i18n.translate('xpack.siem.inspect.modal.indexPatternLabel', {
+  defaultMessage: 'Index pattern',
+});
+
+export const INDEX_PATTERN_DESC = i18n.translate(
+  'xpack.siem.inspect.modal.indexPatternDescription',
+  {
+    defaultMessage: 'The index pattern that connected to the Elasticsearch indices.',
+  }
+);
+
+export const QUERY_TIME = i18n.translate('xpack.siem.inspect.modal.queryTimeLabel', {
+  defaultMessage: 'Query time',
+});
+
+export const QUERY_TIME_DESC = i18n.translate('xpack.siem.inspect.modal.queryTimeDescription', {
+  defaultMessage:
+    'The time it took to process the query. ' +
+    'Does not include the time to send the request or parse it in the browser.',
+});
+
+export const REQUEST_TIMESTAMP = i18n.translate('xpack.siem.inspect.modal.reqTimestampLabel', {
+  defaultMessage: 'Request time',
+});
+
+export const REQUEST_TIMESTAMP_DESC = i18n.translate(
+  'xpack.siem.inspect.modal.reqTimestampDescription',
+  {
+    defaultMessage:
+      'The time of the request from the browser to Elasticsearch and back. ' +
+      'Does not include the time the requested waited in the queue.',
```

Comment:
Does it make sense change `requested` to `request` so it reads as follows?

`Does not include the time the request waited in the queue.`

Reply:
I made a bad copy and paste.

Thank you!!! for the good catch